bean-validation/3.0/_index.md

```diff
@@ -13,6 +13,10 @@ Jakarta Bean Validation defines a metadata model and API for JavaBean and method
 * [Jakarta Bean Validation 3.0 TCK](https://download.eclipse.org/jakartaee/bean-validation/3.0/beanvalidation-tck-dist-3.0.0.zip)
 ([sig](https://download.eclipse.org/jakartaee/bean-validation/3.0/beanvalidation-tck-dist-3.0.0.zip.sig),
 [sha](https://download.eclipse.org/jakartaee/bean-validation/3.0/beanvalidation-tck-dist-3.0.0.zip.sha256),
+[pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
+    * Service Release, 2022-06-02 -- [Jakarta Bean Validation 3.0 TCK](https://download.eclipse.org/jakartaee/bean-validation/3.0/beanvalidation-tck-dist-3.0.1.zip)
+([sig](https://download.eclipse.org/jakartaee/bean-validation/3.0/beanvalidation-tck-dist-3.0.1.zip.sig), 
+[sha](https://download.eclipse.org/jakartaee/bean-validation/3.0/beanvalidation-tck-dist-3.0.1.zip.sha256), 
```

Comment:
Why do we need to mention the previous release 3.0.0 release? The service release should be the latest release included in the platform/web profile release.

Reply:
All 'released' versions are normative. A vendor may be in the process of certifying with the previous release and need access. It remains a valid certification artifact.